server/src/main/java/org/opensearch/tasks/TaskCancellationStats.java

```diff
@@ -22,13 +22,19 @@
  */
 public class TaskCancellationStats implements ToXContentFragment, Writeable {
 
+    private final SearchTaskCancellationStats searchTaskCancellationStats;
     private final SearchShardTaskCancellationStats searchShardTaskCancellationStats;
 
-    public TaskCancellationStats(SearchShardTaskCancellationStats searchShardTaskCancellationStats) {
+    public TaskCancellationStats(
+        SearchTaskCancellationStats searchTaskCancellationStats,
+        SearchShardTaskCancellationStats searchShardTaskCancellationStats
+    ) {
+        this.searchTaskCancellationStats = searchTaskCancellationStats;
         this.searchShardTaskCancellationStats = searchShardTaskCancellationStats;
     }
 
     public TaskCancellationStats(StreamInput in) throws IOException {
+        searchTaskCancellationStats = new SearchTaskCancellationStats(in);
```

Comment:
We need to add a version check here as searchTask stats are newly introduced.
`in.getVersion().onOrAfter(Version.V_3_0_0)`